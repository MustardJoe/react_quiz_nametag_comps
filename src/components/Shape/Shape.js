import React from 'react';

function Shape() {
  return (
    <p>this will change the shape (this will not change the shape,
      but it is the shape component)</p>
  );
}

export default Shape;
