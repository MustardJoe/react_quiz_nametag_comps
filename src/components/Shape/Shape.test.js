import React from 'react';
import { shallow } from 'enzyme';
import Shape from '../Shape/Shape';

describe('Shape component', () => {
  it('renders Shape', () => {

    const wrapper = shallow(<Shape />);
    expect(wrapper).toMatchSnapshot();
  });
});

