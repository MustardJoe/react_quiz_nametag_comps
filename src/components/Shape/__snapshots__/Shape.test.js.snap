// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shape component renders Shape 1`] = `
<p>
  this will change the shape (this will not change the shape, but it is the shape component)
</p>
`;
